live-examples/js-examples/date-tolocaletimestring.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var event = new Date('Tue, August 19, 1975 23:15:30');
+
+console.log(event.toLocaleTimeString('en-US'));
+// expected output: 11:15:30 PM
+
+console.log(event.toLocaleTimeString('it-IT'));
+// expected output: 23:15:30
+
+console.log(event.toLocaleTimeString('ar-EG'));
+// expected output: ١١:١٥:٣٠ م
+
+// NOTE: THERE SEEMS TO BE SOME INCONSISTENCY BETWEEN BROWSERS HERE!
```

Comment:
I don't think we should include this note. If it works across most browsers, we should just ship with the example and rely on the browser compat data to tell people when it doesn't work. If there's clear inconsistency between browsers, we might want to do something like:

```
// expected output (Chrome): XYZ
// expect output (Firefox): ABC
```

Reply:
OK

Reply:
@wbamberg any chance you could check that you are seeing the same issue? I think Chrome returns in UTC. For me it logs 10:15:30 whereas Firefox logs 11:15:30.

Reply:
Sorry - just saw your reply further up. Even setting the time as GMT, Chrome is always an hour behind :/

Reply:
Does it make a difference if you do something like:

`console.log(event.toLocaleTimeString('en-US'), {timeZone: 'UTC'});`

? Otherwise, I'd be tempted to say something hand-wavy like: "the exact output is dependent on your timezone and local configuration".

Reply:
I wondered if you'd tried the above?